pkg/scorecard_result.go

```diff
@@ -14,6 +14,7 @@
 
 package pkg
 
+//nolint:gci
```

Comment:
Fix GCI lint issues. You need to format it this way:

```
import (
<native Golang package imports>
<newline>
<non-Scorecard package imports>
<newline>
<Scorecard package imports>
)
```

Reply:
my bad. Forgot to remove this. Thanks for catching it!